stdlib/public/Concurrency/DispatchGlobalExecutor.inc

```diff
@@ -245,20 +310,34 @@ static void swift_task_enqueueGlobalWithDeadlineImpl(long long sec,
   job->SchedulerPrivate[Job::DispatchQueueIndex] =
       DISPATCH_QUEUE_GLOBAL_EXECUTOR;
 
-  long long nowSec;
-  long long nowNsec;
-  swift_get_time(&nowSec, &nowNsec, (swift_clock_id)clock);
+  uint64_t deadline = sec * NSEC_PER_SEC + nsec;
+  dispatch_time_t when = clock_and_value_to_time(clock, deadline);
+  dispatch_function_t dispatchFunction = 
+    (dispatch_function_t)&_swift_run_job_leeway;
```

Comment:
`dispatchFunction` should just be sunk into the appropriate path.

Reply:
sounds reasonable - will change